lib/fireblocks/jwt.go

```diff
@@ -12,25 +12,26 @@ import (
 	"github.com/google/uuid"
 )
 
+const expirationSecs = 29 // Must be less than 30sec.
+
 // genJWTToken generates a JWT token for the Fireblocks API.
 func (c Client) genJWTToken(uri string, data any) (string, error) {
 	nonce := uuid.New().String()
 	now := time.Now().Unix()
```

Comment:
This is not easy to test in unit tests. We can have a `now func() time.Time` as argument and then can easily make this testable.